pkg/db/client.go

```diff
@@ -24,6 +24,28 @@ type SqlResult interface {
 	RowsAffected() (int64, error)
 }
 
+type Sqler interface {
+	ToSql() (string, []interface{}, error)
+}
+
+func SqllerFmt(format string, a ...any) Sqler {
+	return sqllerFmt{
+		format: format,
+		a:      a,
+	}
+}
+
+type sqllerFmt struct {
```

Comment:
should this be `sqlerFmt ` or maybe rather `sqlFmt`?

Reply:
going with `sqlFmt`..